src/uct/base/uct_md.h

```diff
@@ -51,6 +51,25 @@
     }
 
 
+#define UCT_MD_ATTR_SET_V2_FIELD(_md_attr_dst, _md_attr_src, _field_name, _field_flag) \
+do { \
+    if ((_md_attr_dst)->field_mask & _field_flag) { \
+        ucs_assert((_md_attr_src)->field_mask & _field_flag); \
+        (_md_attr_dst)->_field_name =  (_md_attr_src)->_field_name; \
+    } \
+} while (0)
+
+
+#define UCT_MD_ATTR_COPY_V2_FIELD(_md_attr_dst, _md_attr_src, _field_name, _field_flag) \
```

Comment:
pls do similar changes as prev macro:
1. add ( ) around _field_flag usage
2. why need both UCT_MD_ATTR_COPY_V2_FIELD and UCT_MD_ATTR_SET_V2_FIELD macros?
 I think memcpy should cover both cases